platform/jobs/edxPlatformUnitCoverage.groovy

```diff
@@ -155,21 +135,21 @@ secretMap.each { jobConfigs ->
             }
             timestamps()
             colorizeOutput('gnome-terminal')
-            if (!jobConfig['open'].toBoolean()) {
-                sshAgent(jobConfig['credential'])
+            if (!jobConfig.open.toBoolean()) {
+                sshAgent('jenkins-worker')
             }
             buildName('#\${BUILD_NUMBER}: \${GIT_REVISION,length=8}')
             // Inject CodeCov token so that public jobs can report coverage
-            if (jobConfig['open'].toBoolean()) {
+            if (jobConfig.open.toBoolean()) {
                 credentialsBinding {
                     string('CODE_COV_TOKEN', 'CODE_COV_TOKEN')
                 }
             }
         }
-       /* Copy Artifacts from test subset jobs with build number UNIT_BUILD_NUM */
+        /* Copy Artifacts from test subset jobs with build number UNIT_BUILD_NUM */
         steps {
             for (buildNum = 1; buildNum < 7; buildNum += 1) {
```

Comment:
@estute @michaelyoungstrom Unrelated to this PR - We will need to remember that this hard-coded value ^ exists if we ever change the # of shards!

Reply:
@jzoldak Good call. Making a note of that